main.css

```diff
@@ -0,0 +1,230 @@
+body {
+  background: #eceef1;
+  font-family: "Prompt", sans-serif;
+  font-family: "Montserrat", sans-serif;
+  font-size: 12px;
+  line-height: 1.5;
+  text-align: left;
+}
+
+.signin-section {
+  display: flex;
+  justify-content: flex-end;
+  padding: 10px 30px;
+}
+
+.signin-info p {
+  color: #0077c5;
+  font-size: 12px;
+  margin: 0;
+}
+
+.form-input p {
+  font-size: 10px;
+  margin: 0;
+  padding: 0.4em 0 0.4em 0.2em;
+  text-align: start;
+}
+
+footer div.terms p {
+  margin: 0;
+}
+
+.signin-info p:first-child {
+  color: #505050;
+}
+
+.sign-btn {
+  margin: 0 0 0 10px;
+}
+
+.sign-in-btn {
+  background: #0077c5;
+  border-radius: 4px;
+  border: 1px solid #4c4c4c;
+  color: white;
+  cursor: pointer;
+  font-size: 1.125em;
+  font-weight: normal;
+  height: 34px;
+  line-height: normal;
+  margin: 0;
+  padding: 0 20px;
+  text-align: center;
+  text-transform: none;
+}
+
+.sign-in-lock-icon {
+  background-image: url(https://plugin.intuitcdn.net/identity-authn-core-ui/images/icon-whitelock@3x-76f33cec.png);
+  background-size: contain;
+  background-repeat: no-repeat;
+  display: inline-block;
+  height: 1em;
+  margin-top: 2px;
+  margin-right: 4px;
+  vertical-align: text-top;
+  width: 0.8em;
+}
+
+/* cp ak 2 */
+
+.main-content {
+  display: flex;
+  justify-content: center;
+  padding: 1em 0 1em 0;
+}
+
+.logos-section {
+  padding: 10px 0;
+}
+
+.logo {
+  background-image: url(https://plugin.intuitcdn.net/identity-authn-core-ui/images/ecosystem_logos_new@2x-a8356d8c.png);
+  background-size: 562px 83px;
+  margin: auto;
+  height: 35px;
+  width: 118px;
+}
+
+.logo-icons-section {
+  /* margin: auto;
+    position: fixed; */
+  height: 30px;
+  width: 300px;
+}
+
+.logo-icons-section img {
+  width: inherit;
+}
+
+.form-body {
+  background: #fff;
+  border: 1px solid #c7c7c7;
+  height: 668px;
+  margin: auto;
+  padding: 30px;
+  width: 340px;
+}
+
+form {
+  text-align: center;
+}
+
+.sub-title {
+  font-size: 24px;
+}
+
+.learn-more {
+  color: #0077c5;
+  margin: 0;
+}
+
+.form-header {
+  padding-bottom: 3em;
+}
+
+.form-header p:nth-of-type(1) {
+  margin-bottom: 0;
+}
+
+.form-input {
+  display: flex;
+  flex-direction: column;
+}
+
+.form-input div {
+  height: 3em;
+}
+
+.form-input label {
+  font-size: 14px;
+  color: #404040;
+  padding: 0 0 0.3em 0.1em;
+  text-align: start;
+}
+
+.form-input input {
+  align-self: center;
+  height: 34px;
+  padding: 0 6px 0 8px;
+  width: 320px;
+}
+
+/* cp am 3 */
+
```

Comment:
It’s better to delete commented code. If you need to review a previous version of your code, use git history